meta/main.yml

```diff
@@ -10,6 +10,9 @@ galaxy_info:
       versions:
         - 6
         - 7
+    - name: Amazon
+      versions:
+        - 2015.09
```

Comment:
There is no need to list Amazon as supported distribution as it's based on RedHat which is already listed.